x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/page_steps/add_integration.tsx

```diff
@@ -19,9 +19,9 @@ import { isVerificationError } from '../../../../../../../../services';
 import type { MultiPageStepLayoutProps } from '../../types';
 import type { PackagePolicyFormState } from '../../../types';
 import type { NewPackagePolicy } from '../../../../../../types';
-import { sendCreatePackagePolicy, useStartServices } from '../../../../../../hooks';
+import { sendCreatePackagePolicy, useStartServices, useUIExtension } from '../../../../../../hooks';
```

Comment:
You can move the type import of `RequestError` which is on line 23 to here on line 22.

Reply:
Also, the `PackagePolicyValidationResults` type import can be moved into the same line with the non-type imports from `../../../services`